src/SFA.DAS.Reservations.Web/Startup.cs

```diff
@@ -122,7 +128,8 @@ public void ConfigureServices(IServiceCollection services)
             services.Configure<IISServerOptions>(options => { options.AutomaticAuthentication = false; });
 
             var reservationsWebConfig = serviceProvider.GetService<ReservationsWebConfiguration>();
-
+            var otherConfig = serviceProvider.GetService<AccountApiConfiguration>();
```

Comment:
I can't see anywhere this variable is used

Reply:
oops! pushed the removal of that bit of debug code!